offchainreporting2/internal/protocol/report_generation_leader.go

```diff
@@ -75,7 +75,13 @@ func (repgen *reportGenerationState) startRound() {
 	repgen.leaderState.readyToStartRound = false
 	var query types.Query
 	{
-		ctx, cancel := context.WithTimeout(repgen.ctx, repgen.config.MaxDurationQuery)
+		maxDurationQuery := repgen.config.MaxDurationQuery
+		if maxDurationQuery == 0 {
+			// Some feeds are configured to zero due to the in-memory implementation of Query being a no-op.
+			// This is incompatible with GRPC/LOOPPs, which short circuits on the cancelled context, so we override the value.
+			maxDurationQuery = 500 * time.Millisecond // TODO from config?
```

Comment:
`100ms` is passing the smoke tests.
```suggestion
			maxDurationQuery = 100 * time.Millisecond // TODO from config?
```
We can likely go even lower.

Reply:
Applied this and rebased over here: https://github.com/smartcontractkit/libocr/pull/68